Documentation/mobile-manual.txt

```diff
@@ -197,146 +233,134 @@ the application back button at the top left of the screen on iOS.
 
 [[S_Add_Dive]]
 == Manually adding a new dive to the dive list
 
 One can manually add a dive to the existing dive list. On the dive list
 tap the "plus" sign in the action bar, or use the
-main menu and tap _Manage dives  ->  Add dive manually_. This opens  a screen that is
+main menu and tap _Dive management  ->  Add dive manually_. This opens  a screen that is
 identical to the editing screen discussed above. When a dive is added manually, you
 cannot directly add a dive profile from a dive computer. However, if you do not use a
 dive computer, the duration, depth and several other bits of information about
 the new dive can be entered. The _Action button_ at the bottom of the screen
-contains a disk symbol. Tap this to save the new dive. To cancel any edits, tap the Android Back Button.
+contains a disk symbol. Tap this to save the new dive. To cancel any edits, tap the _Back_ button.
 The left-hand image below shows a screenshot of a dive being created and the right-hand image shows
 the same dive in _Details View_.
 
 image::mobile-images/Manualdive.jpg["FIGURE: Manual dive entry screen",align="center"]
 
 == Deleting a dive
 
-The _Details View_ of a dive has an Action Bar, including a dustbin. If this is tapped,
+The _Details View_ of a dive has an Action Bar with a dustbin. If this is tapped,
 the dive shown in the _Details View_ is deleted. You have a brief opportunity to undo
 the delete by tapping the grey _Undo_ button in the message that appears at the bottom of
 the screen (see image below).
 
 image::mobile-images/Delete_undo.jpg["FIGURE: Undo delete dive",align="center"]
 
 You can also delete a dive from the dive list by long-pressing a dive
-until a red dustbin appears on the right-hand side (see image below). Tap the dustbin. The dive is
-deleted without asking any confirmation because _Subsurface-mobile_ assumes that the
+until a red dustbin appears on the right-hand side (see image below). Tap the dustbin. The dive is deleted without asking any confirmation because _Subsurface-mobile_ assumes that the
 combination of a long tap on the dive with another tap on the red dustbin is an unambiguous
 instruction to delete the dive.
 
 image::mobile-images/RedDustbin.jpg["FIGURE: delete dive from list",align="center"]
 
-== Copying dive details
-
-To copy selected dive details over to another dive, long-press a dive in the
-divelist. This will show a three icons on the right side of the selected dive
-as shown on the image below. The left image will copy details from the selected
-dive and middle button will paste what has been copied before. Naturally you
-should long-press a different dive after copying the details to perform the
-paste operation.
-
-image::mobile-images/CopyPaste.jpg["FIGURE: copy dive details in list",align="center"]
-
-Currently the details to be copied are hard-coded and include the following
-information:
-
-- divemaster
-- buddy
-- suit
-- tags
-- cylinders
-- weights
-
```

Comment:
This section should not be removed. Unlike I earlier stated, the feature is already in master, but it is not available in the official apps yet. Should be available in APK test builds though. (i also have a PR that implemented UI for this feature, but that is not yet in master.)